cmd/amp-agent/core/logs.go

```diff
@@ -55,7 +55,7 @@ func openLogsStream(ID string, lastTimeID string) (io.ReadCloser, error) {
 // Use elasticsearch REST API directly
 func getLastTimeID(ID string) string {
 	request := strings.Replace(elasticSearchTimeIDQuery, "[container_id]", ID, 1)
-	req, err := http.NewRequest("POST", conf.elasticsearchURL+"/amp-logs/_search", bytes.NewBuffer([]byte(request)))
+	req, _ := http.NewRequest("POST", conf.elasticsearchURL+"/amp-logs/_search", bytes.NewBuffer([]byte(request)))
```

Comment:
It's to do not have build warning. All the amp-agent code is going to be updated deeply, this part will disappear because amp-agent no longer be connected to elasticsearch